### DISCUSSION TITLE
Count the different ore variations statistics into one requirement

### DISCUSSION THREAD

**Discussion by glopp** [Zrips/CMI#6667] [Enhancement]
Would like to see possibility of setting up the counting of blocksmined for normal Ores and their deepslate Ore variation together as one requirement. For example diamond_ore and deepslate_diamond_ore counting towards one requirement.

blocksmined 1000 diamonds ore (diamonds and deepslate diamonds)

**Comment by mrfloris** [comment]
```yaml
  - blocksmined:diamond_ore:30
  - blocksmined:deepslate_diamond_ore:30
```
if you do this, then both diamond ore and deepslate diamond ore, 30 of each .. are required to continue.

You can't set it to be like 'all ores together counted as 100 (99 diamond ore and 1 deepslate..) to continue. it's not a dynamic variable.

**Comment by mrfloris** [comment]
On Discord the example was given:
`blocksmined:[diamond_ore,deepslate_diamond_ore]:100`
Which can make sense, and is a nice suggestion. (and no, it currently doesn't do that).
I wonder how intensive it is to constantly having to calculate these values to see if 15 iron ore, 12 gold oren, 17 deepslate diamodns etc are a total for 100+ ores mined.

**Comment by myavuzokumus** [comment]
I think this needs to be added. @Zrips
